test/lib/find-scaffolds.js

```diff
@@ -58,22 +63,52 @@ const after = () => {
 };
 
 
-test('Find Scaffolds success', assert => {
+test('Find Scaffolds API', assert => {
+  before();
```

Comment:
Yeah that all makes a lot of sense. After I started with tape I began to "purify" my functions and modules as much as possible. Will update this code as well.